src/pages/Design/images/Ui4.js

```diff
@@ -0,0 +1,11 @@
+import u4 from "../../../assets/images/design-page/ui_ux/4.svg";
```

Comment:
You could create a template or layout and using props as a parameter to pass the image could work? 
https://stackoverflow.com/questions/46663371/whats-the-best-way-to-create-a-page-template-in-react

Reply:
Yes I will try